projects/sbb-esta/angular-business/dialog/src/_dialog.scss

```diff
@@ -0,0 +1,174 @@
+@import '../../../angular-core/styles/common/colors';
```

Comment:
Sorry, I meant move this entire file content into appropriate *.component.scss.

Reply:
Expect where variables or mixins can be re-used.

Reply:
I have moved the styles to their respective components wherever possible, only one mixin and a few variables are now left in `_dialog.scss`.